docs/tasks/debug-application-cluster/audit.md

```diff
@@ -208,153 +127,50 @@ rules:
 
-Logged events omit the request and response bodies. The `Request` and
-`RequestResponse` levels are equivalent to `Metadata` for legacy format. This legacy format
-of advanced audit is different from the [Legacy Audit](# Legacy Audit) discussed above, such
-as changes to the method values and the introduction of a "stage" for each event.
+- `--audit-log-path` specifies the log file path, that log backend uses to write
+  audit events. Not specifying this flag disables log backend.
+- `--audit-log-maxage` defined the maximum number of days to retain old audit log files.
+- `--audit-log-maxbackup` defines the maximum number of audit log files to retain.
+- `--audit-log-maxsize` defines the maximum size of the audit log file before it gets rotated.
 
-#### Webhook backend
+### Webhook backend
 
-The audit webhook backend can be used to have [kube-apiserver][kube-apiserver]
-send audit events to a remote service. The webhook requires the `AdvancedAuditing`
-feature flag and is configured using the following command line flags:
+Webhook backend sends audit events to a remote API, which is assumed to be the
+same API as [kube-apiserver][kube-apiserver] exposes. You can configure webhook
+audit backend using the following kube-apiserver flags:
 
-```
---audit-webhook-config-file=/etc/kubernetes/audit-webhook-kubeconfig
---audit-webhook-mode=batch
-```
-
-`audit-webhook-mode` controls buffering strategies used by the webhook. Known modes are:
-
-- `batch` - buffer events and asynchronously send the set of events to the external service.
-- `blocking` - block API server responses on sending each event to the external service.
+- `--audit-webhook-config-file` specifies the path to a file with a webhook
+  configuration. Webhook configuration is effectively a [kubeconfig][kubeconfig].
+- `--audit-webhook-mode` define the buffering strategy, one of the following:
+  - `batch` - buffer events and asynchronously send the set of events to the external service.
+  - `blocking` - block API server responses on sending each event to the external service.
 
 The webhook config file uses the kubeconfig format to specify the remote address of
 the service and credentials used to connect to it.
 
-```
-# clusters refers to the remote service.
-clusters:
-  - name: name-of-remote-audit-service
-    cluster:
-      certificate-authority: /path/to/ca.pem  # CA for verifying the remote service.
-      server: https://audit.example.com/audit # URL of remote service to query. Must use 'https'.
-
-# users refers to the API server's webhook configuration.
-users:
-  - name: name-of-api-server
-    user:
-      client-certificate: /path/to/cert.pem # cert for the webhook plugin to use
-      client-key: /path/to/key.pem          # key matching the cert
-
-# kubeconfig files require a context. Provide one for the API server.
-current-context: webhook
-contexts:
-- context:
-    cluster: name-of-remote-audit-service
-    user: name-of-api-sever
-  name: webhook
-```
```

Comment:
Should we retain the config file sample?

Reply:
We have a kubeconfig file in logstash example.
I am OK to let this go.

Reply:
That's the reason I removed it: an example is already present in another place